web-app/src/client.js

```diff
@@ -1,5 +1,6 @@
 import { start } from '@sapper/app';
-
+import './assets/css/tailwind.css';
+import './assets/css/global.css';
```

Comment:
What do you think about flipping the order of these lines so that the import that gets used is distinguished from the ones that are just there?

```suggestion
import './assets/css/global.css';
import './assets/css/tailwind.css';

import { start } from '@sapper/app';
```

Reply:
I'm not sure what you mean...don't both get used throughout the whole app? I suppose that we could load them in the highest-level `_layout.svelte` instead, if that makes the intent more clear.

Also, doesn't putting global before Tailwind introduce potential specificity/override issues, since Tailwind is broader and global is a set of overrides?

Reply:
Oh, I was just talking about the import that get used by this file (`sapper.start`), rather than just being included to get them loaded (the css)

I didn't realize I'd flipped the order of global and tailwind, you're right that tailwind should come first

Reply:
Ah, got it